web/app/proposal/[id]/results/loading.tsx

```diff
@@ -0,0 +1,111 @@
+'use client';
```

Comment:
How do you intend for the function(s) in this file (e.g. PageLoading()) to be a part of the web app? From where are the functions called?

Reply:
function is now called in the Suspense component in the page.tsx file